default.json

```diff
@@ -77,6 +77,14 @@
       "args": ["free", "-m"],
       "link": "free"
     },
+    {
+      "args": ["journalctl", "--system", "-b"],
+      "link": "journal"
```

Comment:
Can we make this boot.log and prev-boot.log instead of journal? "journal" is a bit too general.

Reply:
Done, I also thought about having `journalctl-boot…` but I guess the purpose of the link is to be short.